### PR TITLE
Added logo && Updated Readme

### PR DESCRIPTION
**Pull request by vaibhav-init**

### CODE REVIEW

**Comment by wthrajat** [comment]
@vaibhav-init Readme update should be postponed till better frontend is added.
Edit your changes to only add an icon for now

**Comment by vaibhav-init** [comment]
okay then, I  will make a PR after making a better Frontend :)

File: junk.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def __init__(self):
         self.root = tkinter.Tk()
 
         # Create menu.
+        self.root.iconbitmap("logo.ico")
```
Comment (wthrajat) [inline junk.py:25]: Did you test this locally?

**Review by wthrajat — APPROVED** [review]

File: junk.py (hunk 1 above)
Comment (vaibhav-init) [reply junk.py:25]: yes

**Pull request closed without merging**